projects/ngx-openlayers/src/lib/interactions/draw.component.ts

```diff
@@ -1,65 +1,71 @@
 import { Component, Input, OnDestroy, OnInit, EventEmitter, Output } from '@angular/core';
-import { interaction } from 'openlayers';
 import { MapComponent } from '../map.component';
+import { Draw } from 'ol/interaction';
+import { Collection, Feature } from 'ol';
+import { Vector } from 'ol/source';
+import GeometryType from 'ol/geom/GeometryType';
+import { Style } from 'ol/style';
+import { DrawEvent } from 'ol/interaction/Draw';
+import { StyleFunction } from 'ol/style/Style';
 
 @Component({
   selector: 'aol-interaction-draw',
   template: '',
 })
 export class DrawInteractionComponent implements OnInit, OnDestroy {
-  instance: interaction.Draw;
+  instance: Draw;
 
   @Input()
   clickTolerance?: number;
   @Input()
-  features?: ol.Collection<ol.Feature>;
+  features?: Collection<Feature>;
   @Input()
-  source?: ol.source.Vector;
+  source?: Vector;
   @Input()
   snapTolerance?: number;
   @Input()
-  type: ol.geom.GeometryType;
+  type: GeometryType;
   @Input()
   maxPoints?: number;
   @Input()
   minPoints?: number;
   @Input()
-  finishCondition?: ol.EventsConditionType;
+  finishCondition?: any; // TODO: ol.EventsConditionType;
```

Comment:
In ol 5.3.0 finishCondition has type Condition from import { Condition } from 'ol/events/condition';

see https://openlayers.org/en/latest/apidoc/module-ol_interaction_Draw.html

Reply:
Same below